apps/web/src/components/milestone-logs/recruitment/Record.tsx

```diff
@@ -98,6 +106,33 @@ export const Record: React.FC<RecordProps> = ({
     }
   };
 
+  const handleDeleteJob = (jobId?: string) => {
+    if (jobId) {
+      deleteJob(jobId);
+    }
+
+    setDeleteModalVisible(false);
+  };
+
+  const deleteButton = () => {
+    return canDelete(authUser?.user_id, added_by?.id) ? (
```

Comment:
It might a rare case but what if users with applicant write access added a job and then they lose the access later? can they still see the delete button?

Reply:
no, it'd be blocked by the role mask